tests/flow/test_bidirectional_traversals.py

```diff
@@ -352,3 +352,22 @@ def test13_multiple_bidirectional_edges(self):
                            ['v3', 'v1'],
                            ['v3', 'v3']]
         self.env.assertEquals(actual_result.result_set, expected_result)
+
+    def test14_triple_edge_type(self):
```

Comment:
Put this test in a file like `test_relation_patterns.py`, as it does not test bidirectional traversals.